client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -14,10 +15,12 @@ interface ParamTypes {
 
 const AdminRequestGroupBrowser: FunctionComponent = () => {
     const { id } = useParams<ParamTypes>();
+    const history = useHistory();
     const [requestGroup, setRequestGroup] = useState<RequestGroup | undefined>(undefined);
     const [numTypes, setNumTypes] = useState(0);
     const [showEditGroupModal, setShowEditGroupModal] = useState(false);
     const [showCreateTypeModal, setShowCreateTypeModal] = useState(false);
+    const [showDeleteGroupModal, setShowDeleteGroupModal] = useState(false);
 
     const query = gql`
         query requestGroup($id: ID) {
```

Comment:
Not specific to this PR, but it doesn't seem like we use some of these query fields e.g. group description, updatedAt since they don't appear to be passed to the RequestGroupForm

(sorry it wouldn't let me commend on that area specifically)

Reply:
Good catch. This was left from back when Redux was being used. Removed the unused parts of the query to speed up the loading.